data/human/sales.txt

```diff
@@ -475,6 +475,8 @@ outfitter "Pirate Outfits"
 	"Gatling Gun"
 	"Bullet Boxes"
 	"Brig"
+	"Slave"
```

Comment:
Hey, doesn't this mean you'll be able to buy slaves on Smuggler's Den and Thule, as well as in IIRC Sargas? There's no way the Free Worlds would permit a slave market to operate out in the open like this. You should probably add a separate outfitter for slave markets (assuming we do keep human beings as purchasable in the outfitter - what about adding a Spaceport mission instead?) and add it to certain pirate worlds instead.